src/app/ui/figures/entity-menu/entity-menu-dialog.html

```diff
@@ -90,14 +90,16 @@
       <ng-container
         *ngIf="data.entity && EntityValueFunction(data.entity.maxHealth) > 0 && (!gameManager.isMonster(data.figure) || !gameManager.toMonster(data.figure).immortal)">
         <span class="item">
-          <ghs-pointer-input class="button" (singleClick)="changeHealth(-1)" [repeat]="true">
+          <ghs-pointer-input class="button" [ngClass]="{'disabled' : data.entity.health + health == 0}"
```

Comment:
generally prefer === for JS falsey data types and such, but I am not sure that is the style in this repo